### PR TITLE
Update snmp keys

### PR DESCRIPTION
**Pull request by fabaff**
**Description:**
Update keys for device tracker.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23678

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by MartinHjelmare — COMMENTED** [review]
There are a couple of more occurrences of `authkey` and `privkey` on the page.

File: source/_components/snmp.markdown [hunk 1]
```diff
@@ -91,12 +91,12 @@ baseoid:
   description: The OID prefix where wireless client registrations can be found, usually vendor specific. It's advised to use the numerical notation. To find this base OID, check vendor documentation or check the MIB file for your device.
   required: true
   type: string
-authkey:
-  description: Authentication key for SNMPv3. Variable privkey must also be set.
+auth_key:
+  description: A"uthentication key for SNMPv3. Variable `priv_key` must also be set."
```
Comment (MartinHjelmare) [inline source/_components/snmp.markdown:95]: The quote got misplaced inside authentication.

Comment (klaasnicolaas) [reply source/_components/snmp.markdown:95]: new PR fot that?

Comment (MartinHjelmare) [reply source/_components/snmp.markdown:95]: It would be good to fix it yes. Is that the question?

Comment (klaasnicolaas) [reply source/_components/snmp.markdown:95]: yes that was the question, if it need to be adjusted then just in a new PR 🤣

**Pull request merged**